package.json

```diff
@@ -37,6 +37,7 @@
     "mochawesome": "^7.1.3"
   },
   "dependencies": {
-    "axios": "^0.27.2"
+    "axios": "^0.27.2",
+    "isomorphic-fetch": "^3.0.0"
```

Comment:
install isomorphic-fetch as devDependency

Reply:
This was already set up